### ISSUE TITLE
Move to "import" broke things

### ISSUE THREAD

**Issue by mesoic** [sindresorhus/log-symbols#28]
typescript compiles fine, but I get these errors when I run. I am not willing to change `package.json` to make this work.

```
node:internal/modules/cjs/loader:1109
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
      ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /home/mesoic/Desktop/chainlink/node-operator-management/pegswap/node_modules/log-symbols/index.js
require() of ES modules is not supported.

```

**Comment by sindresorhus** [comment]
Please read the release notes.

**Comment by XingXiaoWu** [comment]
same issuses, but I'm using import

I don't know if it has anything to do with my local configuration
I'm using typescript

```
// package.json 
  "dependencies": {
    "log-symbols": "^5.0.0"
  }
```
```
//tsconfig.json
{
    "compilerOptions": {
      "target": "es5",                            
      "module": "commonjs",                       
      "allowJs": true,                            
      "sourceMap": true,                          
      "outDir": "./bin/",                         
      "strict": true,                             
      "moduleResolution": "node",                 
      "typeRoots": [
        "./node_modules/@types",
        "./typings"
      ],                             
      "esModuleInterop": true,                    
      "skipLibCheck": true,                       
      "forceConsistentCasingInFileNames": true,   
      "resolveJsonModule": true
    }
  }
```

code is 
```
import logSymbols from 'log-symbols';
```

error message
```
internal/modules/cjs/loader.js:1085
      throw new ERR_REQUIRE_ESM(filename, parentPath, packageJsonPath);
      ^

Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /cli/node_modules/log-symbols/index.js
require() of ES modules is not supported.
require() of /cli/node_modules/log-symbols/index.js from /cli/bin/src/utils/messageUtils.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /cli/node_modules/log-symbols/package.json.

    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1085:13)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14)
    at Module.require (internal/modules/cjs/loader.js:957:19)
    at require (internal/modules/cjs/helpers.js:88:18)
    at Object.<anonymous> (/cli/bin/src/utils/messageUtils.js:6:37)
    at Module._compile (internal/modules/cjs/loader.js:1068:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1097:10)
    at Module.load (internal/modules/cjs/loader.js:933:32)
    at Function.Module._load (internal/modules/cjs/loader.js:774:14) {
  code: 'ERR_REQUIRE_ESM'
}
```

**Comment by XingXiaoWu** [comment]
my node version >12, it's 14.17.0

**Comment by XingXiaoWu** [comment]
The `module ` in `tsconfig.json` change the build, using 'esNext' can work, but other node modules can't use import/export. But the new version of log-symbols sets the `type:module`.

 It seems that until other Node libraries allow import/export, the only way to do this is to use the old version.

Or is there a better way?

**Comment by trobert2** [comment]
I don't think this issue should have been closed.

After reading the release notes we get:
> This package is now pure ESM. Please read this.
After following the link we get to https://gist.github.com/sindresorhus/a39789f98801d908bbc7ff3ecc99d99c

even after following the instruction there, I could not get my package to work as ESM. Even if one can afford to completely re-vamp their project and move away from CommonJS and assuming all the libraries you have work with the change (not all will), there are cases where this does not work.
If you are using this tool to build a CLI for example, then new package won't be installed as an executable when using `-g`.

I think at least bug fixes should be ported to `v4` as it works with CommonJS.
I ended up installing `"log-symbols": "4.1.0"`
